### DISCUSSION TITLE
MiniGPT 4 is out

### DISCUSSION THREAD

**Discussion by fuleinist** [oobabooga/text-generation-webui#1437] [General]
https://github.com/Vision-CAIR/MiniGPT-4

Looking forward to get the 4bit model loaded up in webui!

**Comment by Ph0rk0z** [comment]
After https://github.com/Vision-CAIR/MiniGPT-4/blob/main/PrepareVicuna.md

You can quantize it to 4bit.. but then I think you would have to integrate it with GPTQ in their repo. There is currently no integration with image classifiers here afaik.
